scripts/v1alpha2/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+PREFIX="katib"
+CMD_PREFIX="cmd"
+
+SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/../..
+
+cd ${SCRIPT_ROOT}
+
+docker build -t ${PREFIX}/katib-controller -f ${CMD_PREFIX}/katib-controller/Dockerfile .
```

Comment:
build script might be same for both versions if same image contains both versions. We can change it later when all components are built

Reply:
ok